docs/references/differences_with_original_sdk_markdown.md

```diff
@@ -0,0 +1,67 @@
+Differences Between the New and Old Python SDKs
+===============================================
+
+Packaging
+---------
+
+The original [Alteryx Python
+SDK](https://help.alteryx.com/current/developer-help/python-engine-sdk)
+was built around a Python package called `AlteryxPythonSDK`. This
+package is available at runtime inside of Alteryx Designer, and can be
+accessed via import.
+
+The new Python SDK is a standalone Python `pip` package that can be
+installed via `pip install`. It doesn\'t depend on any special libraries
+that only ship with Designer. Given that it is now standalone, we took
+measures to allow tool development outside of Designer.
+
+Getting Started
+---------------
+
+Getting started in the old SDK typically means looking at an example
+plugin, copying the code and file structure, and then modifying it to
+suit your own needs. The new SDK provides a command-line interface (CLI)
+that takes care of all of this project setup. Access the documentation
+[here](https://help.alteryx.com/developer-help/ayx-plugin-cli-quickstart).
```

Comment:
TW review:

Getting started in the old SDK typically means looking at an example
plugin, copying the code and file structure, and then modifying it to
suit your own needs. The new SDK provides a command-line interface (CLI)
that takes care of all of this project setup. Go to the [documentation](https://help.alteryx.com/developer-help/ayx-plugin-cli-quickstart) for more information.

Reply:
fixed